api/src/main/java/org/openmrs/module/reporting/report/renderer/ReportTemplateRenderer.java

```diff
@@ -222,17 +222,22 @@ public String getExpressionSuffix(ReportDesign design) {
 	 * @return the value for the report template replacement given the initial value
 	 */
 	public Object getReplacementValue(Object initialValue) {
+
 		Object replacementValue = "";
-		if (initialValue != null) { 
-			if (initialValue instanceof Cohort) {

```

Comment:
You still need to handle the Cohort case.  Why did you remove this check on Cohort?

Reply:
This was an oversight, let me add that case also

Reply:
The cohort case has been added